build.gradle

```diff
@@ -27,17 +27,17 @@ allprojects {
     repositories {
         google()
         jcenter()
-        maven {
-            credentials {
-                username "$USERNAME"
-                password "$PASSWORD"
-            }
-            url config.duoSdkUrl
-            name 'SurfaceDuo-libraries'
-            authentication {
-                basic(BasicAuthentication)
-            }
-        }
+//        maven {
+//            credentials {
+//                username "$USERNAME"
+//                password "$PASSWORD"
+//            }
+//            url config.duoSdkUrl
+//            name 'SurfaceDuo-libraries'
+//            authentication {
+//                basic(BasicAuthentication)
+//            }
+//        }
```

Comment:
In case we want to keep this commented in the gradle file, maybe add another comment describing what is it used for?